### PR TITLE
chore: regroup docs menu and add specific styling sections with colors, tailwindcss and table sections

### PR DESCRIPTION
**Pull request by colinscz**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

- Document the defined css color variables inside the six-webcomponents library. 
- Regroup the menu section and also add a section for tailwindcss

grafik

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] It's submitted to the `main` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] I have updated the documentation accordingly.
- [x] All tests are passing
- [ ] New/updated tests are included
- [ ] I have updated the "upcoming" section inside _docs/changelog.md_ explaining the changes I contributed

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by stefanrufer** [comment]
Great, thanks. Please add new sub-menu "Styling" and move Tables+Colors there.

File: docs/guide/colors.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+| `--six-color-success-500`                                | <span style="font-size:80px;color:#6ae300">&#9679;</span>                | `#6ae300` |
+| `--six-color-success-light-to-be-defined`                | <span style="font-size:80px;color:#aaff6c">&#9679;</span>                | `#aaff6c` |
+| `--six-color-action-600`                                 | <span style="font-size:80px;color:#1f87e5">&#9679;</span>                | `#1f87e5` |
+| `--six-color-action-500`                                 | <span style="font-size:80px;color:#2196f3">&#9679;</span>                | `#2196f3` |
+| `--six-color-action-light-to-be-defined`                 | <span style="font-size:80px;color:#a8d0f5">&#9679;</span>                | `#a8d0f5` |
+| `--six-color-inactive`                                   | <span style="font-size:80px;color:#c3c3c3">&#9679;</span>                | `#c3c3c3` |
+| `--six-focus-ring-color` based on `six-color-action-600` | <span style="font-size:80px;color:#1f87e5">&#9679;</span>                | `#1f87e5` |
+
+teasdasfdasdf
```
Comment (pennal) [inline docs/guide/colors.md:42]: Shouldn't be here?

**Review by pennal — COMMENTED** [review]
Could you please add the section about tailwind? So we have a sample of how to use the colors with the tailwind config file

File: docs/guide/colors.md (hunk 1 above)
Comment (colinscz) [reply docs/guide/colors.md]: just removed it - thanks

**Review by pennal — APPROVED** [review]

**Pull request merged**